content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,400 @@
+            "productpage": true
+          },
+          "rate": "0.667",
+          "rateOut": "1.334"
+        }
+      },
+      {
+        "data": {
+          "id": "c219903556c3afdb05eda7e610aba628",
+          "nodeType": "app",
+          "namespace": "bookinfo",
+          "app": "ratings",
+          "destServices": {
+            "ratings": true
+          },
+          "rate": "0.442",
+          "rateTcpSentOut": "94.452"
+        }
+      },
+      {
+        "data": {
+          "id": "37ddc91db761d432f3fff1943802cad7",
+          "nodeType": "app",
+          "namespace": "bookinfo",
+          "app": "reviews",
+          "destServices": {
+            "reviews": true
+          },
+          "rate": "0.667",
+          "rateOut": "0.442"
+        }
+      },
+      {
+        "data": {
+          "id": "b30b0078325bf2e1adb4d57c4c0c2665",
+          "nodeType": "unknown",
+          "namespace": "unknown",
+          "workload": "unknown",
+          "app": "unknown",
+          "version": "unknown",
+          "rateOut": "0.667",
+          "isRoot": true
+        }
+      }
+    ],
+    "edges": [
+      {
+        "data": {
+          "id": "36835c5c95ccd1ec98f91cab3577e9f5",
+          "source": "2c22af42b0c750749399ed2838c56054",
+          "target": "35533a08d948509abf8ae4d5d5647594",
+          "rate": "0.343",
+          "rate5XX": "0.343",
+          "percentErr": "100.000",
+          "percentRate": "25.712",
+          "responseTime": "0.000"
+        }
+      },
+      {
+        "data": {
+          "id": "ff5217a9064e30e4fb875256dab56037",
+          "source": "2c22af42b0c750749399ed2838c56054",
+          "target": "37ddc91db761d432f3fff1943802cad7",
+          "rate": "0.667",
+          "percentRate": "50.000",
+          "responseTime": "0.052"
+        }
+      },
+      {
+        "data": {
+          "id": "89fa162a49acca6ff974afd30aab2ff0",
+          "source": "2c22af42b0c750749399ed2838c56054",
+          "target": "6cdb3cf3ee9a17772f13b295368e112a",
+          "rate": "0.324",
+          "percentRate": "24.288",
+          "responseTime": "0.005"
+        }
+      },
+      {
+        "data": {
+          "id": "a553e38605904d17c50ab1d0db84f113",
+          "source": "37ddc91db761d432f3fff1943802cad7",
+          "target": "c219903556c3afdb05eda7e610aba628",
+          "rate": "0.442",
+          "responseTime": "0.030"
+        }
+      },
+      {
+        "data": {
+          "id": "efe83e483ada36899c34ef66a7974d31",
+          "source": "b30b0078325bf2e1adb4d57c4c0c2665",
+          "target": "2c22af42b0c750749399ed2838c56054",
+          "rate": "0.667",
+          "responseTime": "0.038"
+        }
+      },
+      {
+        "data": {
+          "id": "06d772ed417cd829b4b6f95a209ed1b5",
+          "source": "c219903556c3afdb05eda7e610aba628",
+          "target": "e96058d658030da2171c18d09086ee77",
+          "tcpSentRate": "94.452"
+        }
+      }
+    ]
+  }
+}
+{{< /text >}}
```

Comment:
I prefer to leave this because it provides the reader with a sample of what the graph JSON actually looks like. Without it, they really don't have any idea of the API's usefulness because they won't know what pieces of information this API provides. It helps illustrate an answer to the question, "What data does this API give me?"

Reply:
done - example output removed 3ca0cca79c9bf56cd1846d6aa4022662684d75d0